selfdrive/car/chrysler/chryslercan.py

```diff
@@ -8,7 +8,7 @@
 def create_lkas_hud(packer, gear, lkas_active, hud_alert, hud_count, lkas_car_model):
   # LKAS_HUD 0x2a6 (678) Controls what lane-keeping icon is displayed.
 
-  if hud_alert == VisualAlert.steerRequired:
+  if hud_alert == (VisualAlert.steerRequired or VisualAlert.ldw):
```

Comment:
This isn't what you want. It's going to evaluate `(VisualAlert.steerRequired or VisualAlert.ldw)`
```
if hud_alert in [VisualAlert.steerRequired, VisualAlert.ldw]:
```